cmd/podman/pod_unpause.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	podUnpauseFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "unpause all running pods",
+		},
+		LatestFlag,
+	}
+	podUnpauseDescription = `
+   podman pod unpause
+   Unpauses one or more pods.  The pod name or ID can be used.
+`
+
+	podUnpauseCommand = cli.Command{
+		Name:                   "unpause",
+		Usage:                  "Unpause one or more pods",
+		Description:            podUnpauseDescription,
+		Flags:                  podUnpauseFlags,
+		Action:                 podUnpauseCmd,
+		ArgsUsage:              "POD-NAME [POD-NAME ...]",
```

Comment:
Could be Name or ID no?  I think we've other usage's that show you could do one or the other, emulate those.